sources/operator.1.5.0.pxc-operator.json

```diff
@@ -7,14 +7,26 @@
         "pxc": {
           "8.0.19-10.1": {
             "image_path": "percona/percona-xtradb-cluster:8.0.19-10.1",
+            "image_hash": "1058ae8eded735ebdf664807aad7187942fc9a1170b3fd0369574cb61206b63a",
+            "status": "recommended",
+            "critical": false
+          },
+          "8.0.18-9.3": {
+            "image_path": "percona/percona-xtradb-cluster:8.0.18-9.3",
             "image_hash": "7025f539c3fe4b7ca9ba280c944f39da85bfb680080243480d5a13961fc81554",
+            "status": "available",
+            "critical": false
+          },
```

Comment:
we are not going to support 8.0.18, reason - garbd changes was added into 8.0.19 only

Reply:
ok, I have removed it